pkg/telemetry/rtt.go

```diff
@@ -0,0 +1,115 @@
+package telemetry
+
+import (
+	"math"
+	"time"
+
+	"github.com/pion/rtcp"
+)
+
+const (
+	NSUS                 = 1000
+	microssec      int64 = 1000000
+	kNtpJan1970Sec int64 = 2208988800
+)
+const kMaxCompactNtp uint32 = 0xFFFFFFFF
+const kCompactNtpInSecond int64 = 0x10000
+const kNtpInSecond int64 = 1 << 32
+
+// SaturatedUsToCompactNtp convert us to rtp compact ntptime field (from webrtc)
+func SaturatedUsToCompactNtp(us int64) uint32 {
+	if us <= 0 {
+		return 0
+	}
+	if us >= int64(kMaxCompactNtp)*microssec/kCompactNtpInSecond {
+		return kMaxCompactNtp
+	}
+	// To convert to compact ntp need to divide by 1e6 to get seconds,
+	// then multiply by 0x10000 to get the final result.
+	// To avoid float operations, multiplication and division swapped.
+	half := (microssec - 1) / 2
+	quot := us * kCompactNtpInSecond / microssec
+	remain := us * kCompactNtpInSecond % microssec
+	if remain > half {
+		return uint32(quot + 1)
+	}
+	return uint32(quot)
+}
+
+// TimeMicrosToNtp convert us to NtpTime (from webrtc)
+func TimeMicrosToNtp(time_us int64) *NtpTime {
+	// Since this doesn't return a wallclock time, but only NTP representation
+	// of rtc::TimeMillis() clock, the exact offset doesn't matter.
```

Comment:
`rtc::TimeMillis` reference, is that valid?

Reply:
these code are used by me before,  are extracted from webrtc's RR handle, convert C++ to go, I think these comment can be all removed. feel free to change this.

Reply:
removed